start

```diff
@@ -30,12 +32,12 @@ if [ $(id -u) = 0 ]; then
   fi
 
   if [[ $(stat -c "%u" /data) != $UID ]]; then
-    echo "Changing ownership of /data to $UID ..."
+    log "Changing ownership of /data to $UID ..."
     chown -R ${runAsUser}:${runAsGroup} /data
   fi
 
   if [[ ${SKIP_NSSWITCH_CONF^^} != TRUE ]]; then
-    echo 'hosts: files dns' > /etc/nsswitch.conf
+    log 'hosts: files dns' > /etc/nsswitch.conf
```

Comment:
This needs to remain an `echo` ...that was a good call not to do the alias'ing.

Reply:
Good thing you catched this. Silly mistake from my part, and god thank PRs. I'll make a fix for it.